### DISCUSSION TITLE
Update

### DISCUSSION THREAD

**Discussion by Mickael81** [maxrave-dev/SimpMusic#613] [General]
Hello,
The latest version of the application (v.0.2.4) dates back to November 10.
Is a new update going to be released soon?

**Comment by maxrave-dev** [comment]
I try to migrate the app to Jetpack Compose quickly. My plan is to update in last December

**Comment by Mickael81** [comment]
All right! Thank you very much for your feedback and for these explanations. 🙂👍

**Comment by RyzenXVO5** [comment]
Trên Sony 1 III bị tình trạng "Quá thời gian, không thể kết nối Internet" ạ

**Comment by NguyenAnhTruong24** [comment]
Trên các thiết bị Android xuất hiện thông báo lỗi "Quá thời gian, kiểm tra kết nối internet". Mong admin hướng dẫn hay update khắc phục ạ.

**Comment by Mickael81** [comment]
@maxrave-dev Since the latest app update was released (v.0.2.7), I still get the notification message that appears on the screen every time I open SimpMusic. The app works but this bug is quite annoying. 
I also noticed that on some albums, it's the music video that is played instead of the song. The quality and duration are not necessarily the same in both cases.
![Screenshot_20241227_231717_SimpMusic](https://github.com/user-attachments/assets/69c94a93-03f4-4425-bb15-a265731f7006)

**Comment by Mickael81** [comment]
@maxrave-dev I have uploaded a new translation by making a pull request on GitHub. I don't know if everything is ok with Crowdin. So, I hope you will be able to release a new update as soon as possible.
